doc/bucket-replication.md

```diff
@@ -0,0 +1,82 @@
+[NooBaa Operator](../README.md) /
+
+# Bucket Replication
```

Comment:
I would add a definition of the source bucket and the destination/target bucket at the beginning.